### PR TITLE
Adjust other support pages

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: orm/support.adoc [hunk 1]
```diff
@@ -5,14 +5,8 @@
...
+Hibernate ORM is a fully supported component of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
 
-* http://www.redhat.com/products/jbossenterprisemiddleware/application-platform/[Red Hat JBoss Enterprise Application Platform]
-* http://www.redhat.com/products/jbossenterprisemiddleware/web-server/[Red Hat JBoss Enterprise Web Server]
-* http://www.redhat.com/products/jbossenterprisemiddleware/portal/[Red Hat JBoss Portal ]
-* http://www.redhat.com/products/jbossenterprisemiddleware/data-grid/[Red Hat JBoss Data Grid]
-* http://www.redhat.com/products/jbossenterprisemiddleware/business-rules/[Red Hat JBoss BRMS]
-* http://www.redhat.com/products/jbossenterprisemiddleware/soa/[Red Hat SOA Platform]
-* http://www.redhat.com/products/jbossenterprisemiddleware/developer-studio/[Red Hat JBoss Developer Studio]
+Using JBoss EAP, Quarkus, Open Liberty or Spring Boot? They are all part of Red Hat Runtimes.
```
Comment (gsmet) [inline orm/support.adoc:10]: I wasn't sure we should keep this list? Most of the pages redirect to the Middleware homepage now. And I thought support really made sense as part of the Red Hat Runtimes offering.

But happy to reconsider if someone has concerns (and ideas about how to word it)

File: search/support.adoc [hunk 2]
```diff
@@ -5,12 +5,14 @@
...
+Hibernate Search is a fully supported component of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
 
-In addition the http://www.redhat.com/en/technologies/jboss-middleware/data-grid[Red Hat JBoss Data Grid] provides optional extensions such as high performance real time replication for Hibernate Search powered applications running on the Red Hat JBoss Enterprise Application Platform.
+Using JBoss EAP, Quarkus, Open Liberty or Spring Boot? They are all part of Red Hat Runtimes.
 
-The Red Hat JBoss Data Grid also embeds a custom edition of Hibernate Search and allows using most of the same API to run high performance full-text queries and data mining on the data grid.
+In addition, the https://www.redhat.com/en/technologies/jboss-middleware/data-grid[Red Hat Data Grid] provides optional extensions such as high performance real time replication for Hibernate Search powered applications running on one of the Red Hat supported runtime.
+
+The Red Hat Data Grid also embeds a custom edition of Hibernate Search and allows using most of the same API to run high performance full-text queries and data mining on the data grid.
```
Comment (gsmet) [inline search/support.adoc]: Not totally happy of how this turned out. @yrodiere could you have a look and adjust as you see fit?

File: search/support.adoc [hunk 3]
```diff
@@ -5,12 +5,14 @@
...
 
-Hibernate Search is available as an enterprise ready supported library as a component of the http://www.redhat.com/en/technologies/jboss-middleware/application-platform[Red Hat JBoss Enterprise Application Platform].
+Hibernate Search is a fully supported component of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
 
-In addition the http://www.redhat.com/en/technologies/jboss-middleware/data-grid[Red Hat JBoss Data Grid] provides optional extensions such as high performance real time replication for Hibernate Search powered applications running on the Red Hat JBoss Enterprise Application Platform.
+Using JBoss EAP, Quarkus, Open Liberty or Spring Boot? They are all part of Red Hat Runtimes.
 
-The Red Hat JBoss Data Grid also embeds a custom edition of Hibernate Search and allows using most of the same API to run high performance full-text queries and data mining on the data grid.
+In addition, the https://www.redhat.com/en/technologies/jboss-middleware/data-grid[Red Hat Data Grid] provides optional extensions such as high performance real time replication for Hibernate Search powered applications running on one of the Red Hat supported runtime.
+
```
Comment (yrodiere) [inline search/support.adoc]: I'd remove this. We're removing the Infinispan directory provider in Hibernate Search 6, and Infinispan dropped it in version 11, so RHDG might follow pretty soon.

```suggestion
```

File: search/support.adoc [hunk 4]
```diff
@@ -5,12 +5,14 @@
 [[supported-versions]]
 == Supported Products pass:[<i class="icon-user-md icon-fixed-width icon-2x"></i>]
 
-Hibernate Search is available as an enterprise ready supported library as a component of the http://www.redhat.com/en/technologies/jboss-middleware/application-platform[Red Hat JBoss Enterprise Application Platform].
+Hibernate Search is a fully supported component of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
```
Comment (yrodiere) [inline search/support.adoc]:
```suggestion
Hibernate Search is a fully supported component of Red Hat JBoss EAP as part of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
```

File: search/support.adoc (hunk 2 above)
Comment (yrodiere) [reply search/support.adoc]: They are in the process of removing Hibernate Search APIs from Infinispan, so I'd go with this:

```suggestion
[Red Hat Data Grid](https://www.redhat.com/en/technologies/jboss-middleware/data-grid) also uses a custom edition of Hibernate Search internally to provide high performance full-text queries and data mining on the data grid.
```

**Review by yrodiere — CHANGES_REQUESTED** [review]

**Pull request merged**